cmd/dep/status.go

```diff
@@ -827,10 +834,41 @@ func (pi packageImporters) TabString() string {
 	return buf.String()
 }
 
+// projectConstraints is a slice of projectConstraint
+type projectConstraints []projectConstraint
+
+func (pcs projectConstraints) TabString() string {
+	var buf bytes.Buffer
+	w := bufio.NewWriter(&buf)
+
+	// Create a list of constraints string to be printed and sort them.
+	var constList []string
+	for _, pc := range pcs {
+		constList = append(constList, pc.String())
+	}
+	sort.Strings(constList)
```

Comment:
This sorting would be simplified and improved with `projectConstraint` sort by Project implemented in #1413 .